src/ed25519.rs

```diff
@@ -90,13 +90,13 @@ pub fn verify_batch(
     messages: &[&[u8]],
     signatures: &[Signature],
     public_keys: &[PublicKey],
-) -> Result<(), SignatureError>
-{
-    const ASSERT_MESSAGE: &'static [u8] = b"The number of messages, signatures, and public keys must be equal.";
-    assert!(signatures.len()  == messages.len(),    ASSERT_MESSAGE);
-    assert!(signatures.len()  == public_keys.len(), ASSERT_MESSAGE);
-    assert!(public_keys.len() == messages.len(),    ASSERT_MESSAGE);
- 
+) -> Result<(), SignatureError> {
+    const ASSERT_MESSAGE: &'static [u8] =
```

Comment:
Mixing automated reformatting with commits makes the actual changes you're making very hard to review and is generally considered bad practice in open source development.